backend/Program.cs

```diff
@@ -54,6 +56,13 @@
 
 var app = builder.Build();
 
+using (var scope = app.Services.CreateScope())
```

Comment:
I don't think this should be happening in program.cs because it will run the seed data when we deploy this app to production and we should only have this run when we're developing the app on our local machines.

There should be a way to program this so that we can run a command to seed the data when we're setting up the project - I think this was also done in one if our previous exercises.